README.md

```diff
@@ -0,0 +1,115 @@
+use_frameworks!
+
+target '<target name>' do
+  pod 'C0T0', '~> 0.1.0'
+end
+
+```
+
+Then, run the following command:
+
+```bash
+$ pod install
+```
+
+## Usage
+
+### Making a simple GET request
+
+Create an object of `SessionConfiguration` with specified host.
+
+If you're interested in logging requests, responses and errors, just enable logging in `SessionConfiguration` `init` method. By default logging is disable (also the default value is set so you can skip it in init method).
+
+```swift 
+    private let sessionConfiguration = SessionConfiguration(host: "https://api.github.com",
+                                                            loggingEnabled: true)
+```
+
+Next create instance of `ApiService`
+```swift
+    private lazy var apiService = ApiService(configuration: sessionConfiguration)
+```
+
+Remember to retain the `apiService` object instead of creating local variable inside a function's scope, otherways your object may be released before the response will be received.
+
+Here's an example of `requestRepositories(for ogranization)` function with setting repositories array for success or showing the error alert in case of failure. For parsing the results we're using `Codable` models. So for the Result you have to provide a generic type which is conforming to `Codable` protocol.
+
+```swift
+private func requestRepositories(for organization: String) {
+        let apiRequest = ApiRequest(endpoint: "/orgs/\(organization)/repos", method: .GET)
+        
+        apiService.send(request: apiRequest) { [weak self] (result: Result<[Repository], ApiError>) in
+            result.ifSuccess { repositories in
+                self?.repositories = repositories
+            }.else(closure: { error in
+                DispatchQueue.main.async {
+                    self?.showAlertWith(error)
+                }
+            })
+        }
+    }
+```
+
+### URL encoded parameters
+
+If your request needs an URL encoded parameters just create the request like below: 
+```swift
+        let apiRequest = ApiRequest(endpoint: "/endpoint",
+                                    method: .GET, 
+                                    urlParameters: ["sampleKey": "sampleValue"])
+```
+
+### Making a POST request with headers
+
+If your request needs JSON encoded parameters you can achieve that in that way:
+
+```swift
+        let postJSON = """
+                        {
+                            "name": "ToopLoox team member"
+                            "job" : "iOS Developer"
+                        }
+                        """
+        let postJSONData = postJSON.data(using: .utf8)
+        let apiRequest = ApiRequest(endpoint: "/users",
+                                    method: .POST,
+                                    httpBody: postJSONData,
+                                    headers: ["Content-Type": "application/json"])
+       
+```
+
+### Download content
+
+`ApiService` supports also downloading the content. 
```

Comment:
I would swap the order of words `also` and `supports`, imo "ApiService also supports downloading" sounds more natural.